docs/devices/E1766.md

```diff
@@ -54,7 +54,7 @@ The [binding](../guide/usage/binding.md) functionality of this remote varies per
 - 23.0.75 and greater: supports binding to devices only; Also in this case, as for every battery zigbee device that sleep to save battery, before to send bind command from Z2M you have to press a button on remote control to wake up it and be sure remote control receive Z2M binding command.
 
 ### Starting OTA updates
-Press the button on it right before hitting the update button in Z2MA, and again every second or two for about 5 seconds until upgrade begin.
+Press either the open or close button right before hitting the update button in Z2MA, and again every second or two for about 5 seconds until upgrade begins.
```

Comment:
By the way, what is Z2MA? Isn't it supposed to be just Z2M? Could you please correct it if you're already here?